R/utils-selectors.R

```diff
@@ -5,11 +5,13 @@ col_menu_UI <- function(id) { # nolint: object_name_linter
 
 col_menu_server <- function(id,
                             data,
+                            subjid_var,
```

Comment:
My impression is that there maybe cases where we want the subjid_var in the menu, so I would not do this change here. `include_func` provides a general mechanism to select a subset of the columns, Would it be possible to use `include_func` to achieve that effect? I think the same is applicable to `choices` argument.

```r
include_func = function(x) {
        (is.factor(x) || is.character(x)) && (x != subjid_var) && (x %in% choices)
      },
```

Reply:
@luffmark! Thanks for the changes, I have left some comments as I think we can make the code a bit clearer. :+1: